src/XmlDocMarkdown.Core/MarkdownGenerator.cs

```diff
@@ -2094,7 +2096,7 @@ private string MakeRelative(string baseUri, string path)
 				// then the file name is the link
 				return b.Segments.Last();
 			}
-			return result;
+			return RelativeLinks ? $"./{result}" : result;
```

Comment:
We shouldn't prefix with `./` if `result` is already relative, e.g. starts with `../`.